app/models/peep.rb

```diff
@@ -0,0 +1,15 @@
+class Peep
+
+  include DataMapper::Resource
+
+  validates_length_of :message, :min => 1, :max => 140
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, length: value` over `validates_length_of`.